lib/measured/base.rb

```diff
@@ -6,10 +6,19 @@ module Measured
   class UnitError < StandardError ; end
 
   class << self
+    def build(&blk)
+      builder = ConversionBuilder.new
+      builder.instance_eval(&blk)
+
+      clazz = Class.new(Measurable)
+      clazz.instance_variable_set(:@conversion, builder.conversion)
```

Comment:
I like this. But this is your secret to removing the private constant.

Consider, in this current state:
```ruby
pry(main)> parent = Measured::Length.superclass
=> Measured::Measurable
pry(main)> class X < parent ; end
=> nil
pry(main)> X.new(:unit, 1)
NoMethodError: undefined method 'unit_or_alias?' for nil:NilClass
from /Users/kevin/source/measured/lib/measured/measurable.rb:8:in 'initialize'
```

You can still always subclass. But doing so puts you in an invalid state. So guard against the `@covnversion` being missing. And use that to enforce the `build` pattern.